Projects/MakeMeLaugh_Server/Assets/Scripts/RelayStateManager.cs

```diff
@@ -12,12 +13,15 @@
 public class RelayStateManager : MonoBehaviour
 {
     private Task _initAndAuthTask;
+    private VisualElement _overlay;
 
     public string JoinCode;
+    public string ClientHostURI = "https://richard-fine.itch.io/the-laugh-hole?secret=jal3L5588KAmK3SErcfq9IpPAw";

```

Comment:
do we want to keep this in hardcoded?

Reply:
It's editable in the Inspector.

Reply:
Oh, you mean because it's visible to people in Github? I'm not too worried, but I suppose we should try and figure out a build pipeline or something which applies it as a secret.